src/plays/drawing-board/shared/FilePanel.tsx

```diff
@@ -0,0 +1,31 @@
+import { saveAs } from "file-saver";
+import { useCanvas } from "../CanvasContext";
+import { getCanvasImage } from "../utils/canvasUtils";
+import React from "react";
+
+export const FilePanel = () => {
+  const canvasRef = useCanvas();
+
+  const exportToFile = async () => {
+    const file = await getCanvasImage(canvasRef.current);
```

Comment:
i guess there would be no case of reject promise coming but if it is then handle the rejected promise also or let us leave it.

Reply:
Yeah no case of reject promise momentarily so we can leave it